CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 | Version | Description |
 |---------|-------------|
+| | Background images and titles of default screens now configured via backgound items system. ||
```

Comment:
backgound

Reply:
Ooh ta! On spelling mistake out, one in...